### ISSUE TITLE
UI  broken on job page

### ISSUE THREAD

**Issue by pkMinhas** [airpair/consult.airpair.com#68]
The UI is broken on the job page. Cannot view the buttons below **'Want this AirPair?'** on Chrome v47.0.2526.106 (Mac 64-bit). 

![screen shot 2016-01-23 at 10 27 19 pm](https://cloud.githubusercontent.com/assets/621774/12531438/89e1012e-c220-11e5-8656-02f40eeaf5a3.png)

**Comment by jkresner** [comment]
What device / resolution?

**Comment by jkresner** [comment]
ok - I see it now. Thank for the heads up.

**Comment by championswimmer** [comment]
Having the same issue

**Comment by jkresner** [comment]
Yup @championswimmer I experienced it too. Unfortunately it won't be something we get to this weekend.
